website/build_plugins.py

```diff
@@ -51,7 +51,7 @@
     'PLUGIN_NAME',
     'PLUGIN_AUTHOR',
     'PLUGIN_VERSION',
-    'PLUGIN_API_VERSIONS',
+    'PLUGIN_API_VERSIONS',  # Determine the version of Picard's API to be used in the plugin
```

Comment:
Umm.. I decided to remove this since the other KNOWN_DATA got no in-line comment here..
Also, the plugin developer won't have any time to look at this comment too
So, yeah I'm removing it :)